sql/driver/conn.go

```diff
@@ -56,7 +56,7 @@ func (c *conn) Prepare(query string) (driver.Stmt, error) {
 }
 
 func (c *conn) Begin() (driver.Tx, error) {
-	c.session = []byte{}
+	c.session = nil
```

Comment:
Ah ha. I assumed this was connection begin. I need to better understand this driver interface.